### ISSUE TITLE
Scala 3 - Can't navigate scala3-library source code

### ISSUE THREAD

**Issue by yankun1992** [scalameta/metals#2662]
**Describe the bug**
The metals **PACKAGES** window can't navigate  scala3-library source code

**To Reproduce**

1. Go to VS Code Metals button.
2. Click on PACKAGES
![图片](https://user-images.githubusercontent.com/10016474/113236031-b1e28080-92d6-11eb-9b79-0278fea9b170.png)
3. Scroll down to scala3-library_3.0.0-RC1.jar
![图片](https://user-images.githubusercontent.com/10016474/113236928-7052d500-92d8-11eb-99d8-47d381a46730.png)

**Installation:**

- Operating system: Windows 10
- Editor: Visual Studio Code
- Metals version: v0.10.0

**Additional context**
I can't navigate the source code of my own project also.
![图片](https://user-images.githubusercontent.com/10016474/113236516-95931380-92d7-11eb-83ae-36cc6c1b65c8.png)

When use TASTy reflect API，can't navigate  it 
![图片](https://user-images.githubusercontent.com/10016474/113236765-194d0000-92d8-11eb-8514-8d6155eaa4e7.png)
The Inlined and Block can't see source code.

**Search terms**
scala3 source code can not be navigated.

**Comment by tgodzik** [comment]
Thanks for reporting! I can confirm this is the case, not sure exactly what is happening, we'll need to investigate.

**Comment by dos65** [comment]
The first problem with `Packages`  happens because TreeView for scala jars is based on `ScalaSignature` annotation.
Scala 3 artifacts don't have it so we need to read this information from Tasty.

This might be blocked by issues related to TastyInspector: [1](https://github.com/lampepfl/dotty/issues/11696), [2](https://github.com/lampepfl/dotty/issues/11697). There might be a workaround by using an alternative way of reading Tasty but I would prefer to wait when TastyInspector will be fixed.


> The Inlined and Block can't see source code.
This is fixed in #2600

**Comment by tgodzik** [comment]
I think using tasty is the right thing to do here, so we should wait for those two issues to get fixed.

**Comment by dos65** [comment]
Close this as a duplicate of https://github.com/scalameta/metals/issues/2859.
The new one is more exact as it describes only one unresolved issue.
